README.md

```diff
@@ -61,29 +61,23 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
 #### Create an app for Mattermost
 
 1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
-2. In the top left select **Develop** and then **Build Legacy App**.
-3. Select **OAuth** in **Choose your app type** section.
-4. Enter a name for your app.
+2. In the top left select **Develop** and then **Build App**.
+3. On top you can choose either the **development** or **production** app. If you wish to publish the app on Zoom Marketplace then please select production. The plugin supports apps that are published in the Zoom Marketplace.
```

Comment:
I don't think the customer will be publishing it to the marketplace. I guess I don't see the use case of the customer publishing the app. Maybe we should just suggest select `Development` and explain why?

Also, is there a risk of choosing `Production`, even if they are not intending to publish to the marketplace? If it's not possible to set up with `Production`, and we can't do anything about that, then we need to say `Development`, but I would prefer to tell the customer to use `Production` if possible.

```suggestion
3. On top choose **Development**. We would choose **Production** if we were publishing to marketplace, but we won't be doing that here.
```